backend/tests/houseblockList.test.js

```diff
@@ -197,7 +197,7 @@ describe('Houseblock list controller', () => {
       triggersIEPWarning: ['UnacceptableAbsence', 'Refused'],
     })
     whereaboutsApi.getAgencyGroupLocations = jest.fn()
-    whereaboutsApi.getAgencyGroupLocations.mockReturnValue([])
+    whereaboutsApi.getAgencyGroupLocations.mockReturnValue([1])
```

Comment:
not sure this looks correct - don't you need something like
```
[ { locationId: 1 } ]
```
instead since the code maps to `locationId` ?

Reply:
guess it doesn't matter though as only test data

Reply:
Yes you're right.  The result gets thrown away by the test though... so I might just drop this in the next PR.